build/ci/evergreen.yml

```diff
@@ -334,6 +334,18 @@ tasks:
     commands:
       - func: "clone"
       - func: "check license header"
+  - name: brew_e2e
+    tags: [ "code_health" ]
+    depends_on:
+      - name: compile
+        variant: "code_health"
+        patch_optional: true
+    commands:
+      - func: "clone"
+      - func: "install gotestsum"
+      - func: "e2e test"
+        vars:
+          E2E_TAGS: brew
```

Comment:
[nit] we should not move e2e to be part of the code health, if your end goal is to have e2e be mandatory that's a different conversation

Reply:
yes, the main reason was to make this test mandatory

Reply:
let's rollback this and discuss with the team making e2e mandatory

Reply:
As part of the discussion, mandatory for a code change vs mandatory for a release (only required in the PCT runs)

Reply:
I have removed this change from the PR. I will open a discussion in the slack channel. I agree that we may just need to make this test mandatory for a release.